styles.css

```
@@ -1,22 +1,54 @@
 * {
-  font-family: 'Arial';
+  font-family: "Arial";
+}
+
+body {
+  text-align: center;
+}
+
+#play-button {
+  padding-left: 1rem;
+  margin-bottom: 1rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-right: 1rem;
```

This padding could be combined into one line, just for cleanliness, but not necessary
```suggestion
  padding: 0.5rem 1rem;
```